__test__/server/db/utils.js

```diff
@@ -0,0 +1,33 @@
+export const tables = [
+  'log',
+  'message',
+  'user_cell',
+  'job_request',
+  // 'migrations' // Now automatically managed by Knex
```

Comment:
Wondering if we should just delete this line? Cause a few months after this PR this becomes a reference to a past state of the code that's no longer relevant? Could be useful though